### PR TITLE
Faq section

### LINKED ISSUE #32: FAQ Expanding Item
Should expand when clicked and collapse when clicked again
Icon should switch between +/-

### PR DESCRIPTION
**Pull request by roball24**
Fixes #32 #41 .

### PR Description
The entire FAQ section

### What needs to be reviewed?
FAQ section and ExpandingItem

### Context?
I would add pics, but you should run this one

### CODE REVIEW

File: app/pages/blackout/header.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ class BlackoutHeader extends React.Component {
             <Container>
                 <HeaderWrapper>
                     <HeaderLogoWrapper>
-                        <HeaderLogo src={HeaderLogoImage}></HeaderLogo>
+                        <HeaderLogo src={HeaderLogoImage} />
```
Comment (connorkrupp) [inline app/pages/blackout/header.jsx]: lol thanks I forgot to fix that

File: app/pages/blackout/hero.jsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { push } from 'react-router-redux';
 const VideoPlaceholderImage = require('../../../static/blackout/video-placeholder.png');
 
 const HeroContainer = styled.div`
+    paddingTop: 75px;
+    paddingBottom: 75px;
```
Comment (connorkrupp) [inline app/pages/blackout/hero.jsx]: why not just padding: 75px 0;

**Comment by connorkrupp** [comment]
screen shot 2017-05-10 at 8 18 55 pm
Would it be easy to fix the spacing on this or is it not worth it

**Comment by connorkrupp** [comment]
The only other thing that seems a bit off is how the expansion pushes down the content, it just seems a bit jagged compared to the smooth expansion, but I don't know how to fix that Other than those two this looks $waggy

**Comment by roball24** [comment]
the question is a bit wonky. mhacks 9 was the same way. I'll try- I think its something a flexbox might fix

**Comment by roball24** [comment]
should be good now

**Pull request merged**